extensions/azurePublishNew/src/components/shared/ChooseResourcesList.tsx

```diff
@@ -155,7 +155,7 @@ export const ChooseResourcesList = (props: Props) => {
 
   return (
     <FocusZone direction={FocusZoneDirection.vertical}>
-      <List items={items} onRenderCell={renderItem} />
+      <List items={[...items]} onRenderCell={renderItem} />
```

Comment:
why are you doing this? it would cause lots of unnecessary re-renders, you already are sending a filtered array to this component which guarantees a new reference every time.

Also, in azureProvisionDialog.tsx L1030 where you use this component, the return value should be null instead of Fragment if you dont want to render anything.

Reply:
ChooseResourceList was not re-rendering on selection change even though the newly selected values are coming in. please suggest if there is a better way.

Reply:
Yeah the Fluent controls don't re-render unless the items array changes.  It doesn't care about each item so you have to either do this or slice()